app/services/clients/adminusers.client.js

```diff
@@ -2,7 +2,8 @@
 
 const lodash = require('lodash')
 
-const baseClient = require('./base-client/base.client')
+const { Client } = require('@govuk-pay/pay-js-commons/lib/utils/axios-base-client/axios-base-client')
```

Comment:
I know it's not related to this PR but I'd suggest avoiding being so specific about the path inside the other project.  I'd personally go with:

```
const { AxiosClient } = require('@govuk-pay/pay-js-commons')
```

Or if you want it kept separate there are ways to have a shorter path e.g.

```
const { Client } = require('@govuk-pay/pay-js-commons/axios')
```

To be really clear:  This would need to be changed in `pay-js-commons` and it's not really part of this PR but it's related to this wider piece of work.

Reply:
Good suggestion. I agree it would be much neater.
I also agree that it's not really part of this PR so I propose creating a new ticket to cover this. It should be applied to other repositories already amended for the new version of `pay-js-commons` containing the new `axios-base-client` namely `pay-products-ui` and `pay-frontend`.

Reply:
I have created a new ticket for this:
https://payments-platform.atlassian.net/browse/PP-12468